### PR TITLE
Update plugins.rst

### PR DESCRIPTION
**Pull request by alexsmonte**
Fiz a tradução completa de plugins

### CODE REVIEW

File: pt/plugins.rst [hunk 1]
```diff
@@ -1,12 +1,647 @@
...
+    Plugin::load('AcmeCorp/ContactManager');
+
+    // Carrega todos os plugins de uma só vez
+    Plugin::loadAll();
+
+``loadAll()`` carrega todos os plugins disponíveis, permitindo que você especifique determinadas
+configurações para plugins. ``load()`` funciona de forma semelhante, mas apenas carrega o
+Plugins que você especifica explicitamente.
+
+    ``Plugin::loadAll()`` não irá carregar os plugins vendor namespaced que não são
```
Comment (markstory) [inline pt/plugins.rst]: Looks like you might be missing an annotation here. Perhaps a `.. note::`?

File: pt/plugins.rst [hunk 2]
```diff
@@ -1,12 +1,647 @@
...
+Por exemplo, vamos supor que você queria usar o plugin do ContactManager
+ContactInfoHelper para produzir algumas informações de contato legítimas em
+uma das suas opiniões. No seu controller, o ``$helpers`` array
+poderia ficar assim ::
+
+    public $helpers = ['ContactManager.ContactInfo'];
+
 .. note::
-    Atualmente, a documentação desta página não é suportada em português.
+	Esse nome de classe separado por pontos é denominado :term: `sintaxe de plugin '.
```
Comment (markstory) [inline pt/plugins.rst]: There shouldn't be a space after `:term:`

File: pt/plugins.rst [hunk 3]
```diff
@@ -1,12 +1,647 @@
...
+
+Você poderia então acessar o ``ContactInfoHelper`` como
+qualquer outro helper em sua view, como ::
+
+    echo $this->ContactInfo->address($contact);
+
+Criando seus próprios complementos
+=========================
```
Comment (markstory) [inline pt/plugins.rst]: Title underlines need to be the same length as the text above them.

File: pt/plugins.rst [hunk 4]
```diff
@@ -1,12 +1,647 @@
...
+Agora você pode user o bake com as mesmas convenções que se aplicam ao resto
+do seu aplicativo. Por exemplo - baking controllers:
+
+.. code-block:: bash
+
+    bin/cake bake controller --plugin ContactManager Contacts
+
+
+Consulte o capítulo
+:Doc: `/bake/usage` se você
```
Comment (markstory) [inline pt/plugins.rst]: This needs to be `:doc:` (all lower case)

File: pt/plugins.rst [hunk 5]
```diff
@@ -1,12 +1,647 @@
...
+        function ($routes) {
+            $routes->get('/contacts', ['controller' => 'Contacts']);
+            $routes->get('/contacts/:id', ['controller' => 'Contacts', 'action' => 'view']);
+            $routes->put('/contacts/:id', ['controller' => 'Contacts', 'action' => 'update']);
+        }
+    );
+
+
+O código acima irá conectar as rotas padrão para o seu plugin. Você pode personalizar isso
+no arquivo com rotas mais específicas mais tarde.
```
Comment (markstory) [inline pt/plugins.rst]: Missing a `::` at the end of this line.

**Comment by alexsmonte** [comment]
Close this I'm going to send it again, I messed it up!

**Comment by markstory** [comment]
Ok.

**Pull request closed without merging**